gallery/src/styles/CssVariables/CssVariablesPage.tsx

```diff
@@ -211,8 +211,8 @@ const CssVariablesPage = () => {
             <ColorDocRow colorVar="--nx-color-border">
               Color for standard higher-level borders
             </ColorDocRow>
-            <ColorDocRow colorVar="--nx-color-subsection-border">
-              Color for standard lower-level borders such as those of tile subsections
+            <ColorDocRow colorVar="--nx-color-border-secondary">
+              Color for lighter borders
```

Comment:
Another place where it says "lighter" and probably shouldn't

Reply:
> Another place where it says "lighter" and probably shouldn't

Thanks: https://github.com/sonatype/sonatype-react-shared-components/pull/444/commits/44327d26832612d170fd268994dbc3a5d004ac1c